### DISCUSSION TITLE
11/21 신명희 블로그 2회차

### DISCUSSION THREAD

**Discussion by mh0223** [Orm-camp/blog-feedback#36] [동글동글 3기 피드백]
<a href="https://velog.io/@mh0223/%ED%81%B4%EB%9E%98%EC%8A%A4%EB%A5%BC-%EB%8D%B0%EC%9D%B4%ED%84%B0-%EC%A0%80%EC%9E%A5%EC%86%8C%EB%A1%9C-%EC%82%AC%EC%9A%A9%ED%96%88%EB%8D%98-%EC%8B%A4%EC%88%98%EC%99%80-%EA%B0%9D%EC%B2%B4-%EC%A7%80%ED%96%A5%EC%A0%81-%EA%B0%9C%EC%84%A0-%EA%B3%BC%EC%A0%95">블로그 주소</a>

피드백 많이 부탁드립니다!!

**Comment by baaanjy** [comment]
저도 최근에 클래스를 써야하는 상황인지 고민했었는데, 정리 잘해주셔서 도움이 될 것 같아요! 👍

**Comment by oratio100** [comment]
앞서 작성한 프로젝트 회고에서 이어지는 흐름이 좋았습니다. 
글이 생각보다 길어서 핵심인 "자, 이제 단계별로 따져보자"로 넘어가는 시간이 오래걸려서 앞의 개념 정리를 조금 간단하게 해주셔도 좋을 것 같습니다.

**Reply by mh0223** [reply]
저두 그 부분이 걸렸었는데 역시,, 말씀해주신대로 더 줄여봐야겠습니다!

**Comment by Orm-camp** [comment]
피드백을 듣고, 이전에 작성했던 코드들을 다시 한 번 들여다보고 수정할 부분까지 찾아내시는 모습이 너무 보기 좋았습니다!
다만 지연님 의견처럼 저도 다음 파트로 넘어가는 데에 있어 시간이 꽤 걸렸다는 느낌이 들었습니다.
오늘 읽은 글처럼 글을 짧고 임팩트 있게 구성해 보는 연습이 함께 된다면 더더욱 좋은 글이 될 거 같아요! 👍🏻

**Comment by kimjigyun** [comment]
생각도 많이 하시구 정리도 잘하시는것 같아요 ! 긴 글 쓰시느라 고생하셨습니다.

**Comment by ksj0621** [comment]
늘 생각하지만 글을 쓰실 때 주제를 잡고 주제를 전개하는 과정이 능숙하시다고 느껴요! 마치 본인의 문제를 파악하고 그걸 해결해나가는 일을 자주 해 본 사람의 사고방식이 드러난달까요!! 다만 저처럼 아는게 없는 초심자가 읽기에는 장벽이 느껴지긴 했는데 아마 타겟 독자층이 있으신게 아닐까 생각은 합니당
